routes/spec.js

```diff
@@ -808,27 +812,28 @@ Currently the API has a rate limit of **60 requests/minute** and **50,000 reques
           },
         },
         route: () => '/skyblock/profile/:player/:profile?',
-        func: (req, res) => {
-          getUUID(req.params.player, (err, uuid) => {
-            if (err) {
-              return res.status(404).json({ error: err });
-            }
-            buildProfile(uuid, req.params.profile, true, (err, profile) => {
-              if (err) {
-                return res.json({ error: err });
+        func: async (request, response) => {
+          try {
+            const uuid = await getUUID(request.params.player);
+            try {
+              // TODO: Update when buildProfile changed
+              const profile = await buildProfile(uuid, request.params.profile);
+              try {
+                const players = await populatePlayers(Object.keys(profile.members).map((uuid) => ({ uuid })));
+                players.forEach((player) => {
+                  profile.members[player.profile.uuid].player = player.profile;
+                });
+                response.json(profile);
+              } catch (error) {
+                response.status(500).json({ error });
               }
-              populatePlayers(Object.keys(profile.members).map((uuid) => ({ uuid })), (err, players) => {
-                if (err) {
-                  res.status(500).json({ error: err });
-                } else {
-                  players.forEach((player) => {
-                    profile.members[player.profile.uuid].player = player.profile;
-                  });
-                  res.json(profile);
-                }
-              });
-            });
-          });
+            } catch (error) {
+              // TODO: Here also
```

Comment:
What is this `TODO` referring to?

Reply:
That was a note to self fixed in https://github.com/slothpixel/core/pull/385/commits/96a7dc6444b8462bdc20a5d5980e0b49c55837ac.